pkg/test/framework/runtime/components/apps/kube.go

```diff
@@ -302,26 +324,62 @@ func (c *kubeComponent) Start(ctx context.Instance, scope lifecycle.Scope) (err
 		return err
 	}
 	c.env = env
+	namespace := env.NamespaceForScope(scope)
+
+	// If test not require their apps explicitly, we deploy a suite of default apps.
+	if len(c.requiredApps) == 0 {
```

Comment:
Is it possible here to do something like:

if len(c.requiredApps) == 0 {
  c.requiredApps = createDefaultApps()
}

And then we are consistent is using the same mechanism for building the apps?

Reply:
Done.